setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
@efiring Any special reason for removing `--doctest-modules`?  I was planning on writing some examples/doctests in the docstrings.  If you think that is a bad idea let me know, so I can invest on a separate test module.